scalability_and_performance/ztp-zero-touch-provisioning.adoc

```diff
@@ -26,3 +25,46 @@ include::modules/ztp-low-latency-for-distributed-units-dus.adoc[leveloffset=+1]
 include::modules/ztp-single-node-clusters.adoc[leveloffset=+1]
 
 include::modules/ztp-the-gitops-approach.adoc[leveloffset=+1]
+
+//planning and prereqs
+
+include::modules/ztp-acm-preparing-to-install-disconnected-acm.adoc[leveloffset=+1]
+
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+Deploying a disconnected registry host based on the `docker.io/library/registry:2` API for {product-title} is not officially supported by Red Hat. You can create a mirror host based on the `docker.io/library/registry:2` API with the following unsupported procedure.
+====
+
+include::modules/installation-creating-mirror-registry.adoc[leveloffset=+1]
+
+[IMPORTANT]
```

Comment:
This note's rendering after the procedure, so it makes less sense. I think that changing the if statements so that you get [this note](https://github.com/openshift/openshift-docs/blame/master/modules/installation-creating-mirror-registry.adoc#L17) (or similar) in the module makes more sense.